source/globalCommands.py

```diff
@@ -3143,6 +3143,11 @@ def script_startWxInspectionTool(self, gesture):
 	)
 	@gui.blockAction.when(gui.blockAction.Context.SECURE_MODE)
 	def script_navigatorObject_devInfo(self, gesture):
+		if log.getEffectiveLevel() == 100:
```

Comment:
I don't think 100 is a valid level, please use `logging.INFO` rather than a magic number

https://docs.python.org/3/library/logging.html#levels